system-addon/lib/TopSitesFeed.jsm

```diff
@@ -366,7 +366,7 @@ this.TopSitesFeed = class TopSitesFeed {
       // All these actions mean we need new top sites
       case at.MIGRATION_COMPLETED:
       case at.PLACES_HISTORY_CLEARED:
-      case at.PLACES_LINKS_DELETED:
+      case at.PLACES_LINKS_CHANGED:
```

Comment:
Ah hrm.. Also we end up unnecessarily refreshing top sites now for bookmark triggered CHANGED now… oh well?

But main issue is if it needs to broadcast anymore, and seems like no?

Reply:
should it still expire the frecentCache?